bom/pom.xml

```diff
@@ -47,6 +47,7 @@
     <javassist.version>3.24.1-GA</javassist.version>
     <jaxb.api.version>2.2.12</jaxb.api.version>
     <javax.annotation.api.version>1.3.2</javax.annotation.api.version>
+    <jakarta.enterprise.cdi-api.version>2.0.2</jakarta.enterprise.cdi-api.version>
```

Comment:
this should already be managed by Quarkus. We import Quarkus' BOM pom so there should be no need to specify it here again ...